app/handler/commands.js

```diff
@@ -685,6 +686,28 @@ exports.handleMessage = function (steamID, message) {
                 client.chatMessage(steamID, 'You are not running the bot with PM2! See the documentation: https://github.com/Nicklason/tf2-automatic/wiki/PM2');
             }
         });
+    } else if (isAdmin && command === 'pricecheck') {
+        const params = getParams(message.substring(command.length + 1).trim());
+
+        if (params.sku === undefined) {
+            const item = getItemFromParams(steamID, params);
+
+            if (item === null) {
+                return;
+            }
+
+            params.sku = SKU.fromObject(item);
+        }
+
+        params.sku = SKU.fromObject(fixItem(SKU.fromString(params.sku)));
+
+        api.requestCheck(params.sku, 'bptf', function (err) {
+            if (err) {
+                client.chatMessage(steamID, 'Failed to add the item to the pricelist: ' + (err.body && err.body.message ? err.body.message : err.message));
+                return;
+            }
+            client.chatMessage(steamID, 'Price update request has been sent, item price updating soon...');
```

Comment:
Change it to price check has been requested, the item will be checked.

The price might not change, and it might take a while for it to be checked because people like @ZeusJunior spam it with skins 😢

Reply:
Only 21000 every other day :))

Reply:
wtf